src/api/unified_api.py

```
@@ -1070,6 +1084,11 @@ async def handle_unified_request(request, api_key: str, source_format: str):
         # Anthropic格式需要max_tokens字段
         if source_format == "anthropic" and not request_data.get("max_tokens"):
             raise HTTPException(status_code=400, detail="max_tokens is required for Anthropic format")
+
+        # Responses格式的基本验证
+        if source_format == "responses":
+            if not request_data.get("input"):
+                raise HTTPException(status_code=400, detail="input is required for Responses format")
```

Ruff (0.13.3)

Abstract `raise` to an inner function

(TRY301)